tests/python/pants_test/reporting/test_reporting_integration.py

```diff
@@ -185,6 +186,35 @@ def test_zipkin_reporter(self):
       self.assertTrue(main_children)
       self.assertTrue(any(span['name'] == 'cloc' for span in main_children))
 
+  def test_zipkin_reporter_with_given_trace_id_parent_id(self):
+    ZipkinHandler = zipkin_handler()
+    with http_server(ZipkinHandler) as port:
+      endpoint = "http://localhost:{}".format(port)
+      trace_id = "0123456789abcdef"
```

Comment:
We should assert that the trace_id and parentId were set properly in the trace our server got :)